src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     * Returns a concatenated version of the printable strings of each object.
```

Comment:
Good that you write the comment, but "concatenated version" doesn't have clear meaning.